<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginConfigWidget</class>
 <widget class="QDialog" name="PluginConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugin configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Plugins</string>
        </property>
        <property name="icon">
         <iconset theme="preferences-plugin">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>6</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="page_plugins">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="layout_information"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layout_plugins">
           <item>
            <layout class="QVBoxLayout" name="layout_allPlugins">
             <item>
              <widget class="QLabel" name="label_allPlugins">
               <property name="text">
                <string>All plugins</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="layout_allPluginsControl">
               <item>
                <widget class="QListWidget" name="listWidget_allPlugins">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="layout_control">
                 <item>
                  <spacer name="verticalSpacer_control1">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_left">
                   <property name="icon">
                    <iconset theme="arrow-right">
                     <normaloff>.</normaloff>.</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_right">
                   <property name="icon">
                    <iconset theme="arrow-left">
                     <normaloff>.</normaloff>.</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_control2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="layout_enabledPlugins">
             <item>
              <widget class="QLabel" name="label_enabledPlugins">
               <property name="text">
                <string>Enabled plugins</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="layout_enabledPluginsControls">
               <item>
                <widget class="QListWidget" name="listWidget_enabledPlugins"/>
               </item>
               <item>
                <layout class="QVBoxLayout" name="layout_order">
                 <item>
                  <spacer name="verticalSpacer_order1">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_up">
                   <property name="icon">
                    <iconset theme="arrow-up">
                     <normaloff>.</normaloff>.</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_down">
                   <property name="icon">
                    <iconset theme="arrow-down">
                     <normaloff>.</normaloff>.</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_order2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
